R/script_generation.R

```diff
@@ -0,0 +1,129 @@
+
+#' Gather and format code to be exported.
+#'
+#' @param meta_action The return from a call to `shinymeta::metaAction()`.
+#' @param expanded_chain The result from a call to `shinymeta::expandChain()`.
+#'
+#' @return Formatted code as text.
+.BuildMetaPairwiseExportCode <- function(meta_action = NULL, expanded_chain = NULL) {
```

Comment:
Are you using the leading . to differentiate functions?

Reply:
This is the standard for "private" functions, ie. those which are not to be called from outside this file.